src/Avalonia.Themes.Fluent/Controls/PathIcon.xaml

```diff
@@ -10,6 +10,7 @@
     </StackPanel>
   </Design.PreviewWith>
   <ControlTheme x:Key="{x:Type PathIcon}" TargetType="PathIcon">
+    <Setter Property="Foreground" Value="{DynamicResource ThemeForegroundColor}" />
```

Comment:
ThemeForegroundColor doesn't exist in Fluent.

Also, we normally leave foreground empty in controls, so it can be inherited.

Reply:
Switched to TextControlForeground

As for inheritance it doesn't work here for some reason